CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # gneiss changelog
+## Version 0.1.0  (changes since 0.1.0 go here)
```

Comment:
If you are tagging the repo as of the merge of this PR, then it may make more sense to remove these two lines, and (below) update the "Version 0.0.2 (changes since 0.0.2 go here)" to "Version 0.0.2".